parm/ioda/bufr2ioda/bufr2ioda_satwind_amv_goes.json

```diff
@@ -1,7 +1,7 @@
 {
   "data_format"      : "bufr_d",
   "data_type"        : "satwnd",
-  "cycle_type"       : "{{ DUMP }}",
```

Comment:
I think I remember @aerorahul saying we should go to `RUN` eventually instead of `DUMP` but perhaps I'm misremembering.

Reply:
Replace `DUMP` with `RUN`.   Done at e9515e0.

g-w sets `RUN` to `gfs` and `gdas` when running gfs and gdas cycle, respectively.  `RUN` is `enkfgfs` or `enkfgdas` when running enkf jobs.  Since g-w `prepatmiodaobs` job is associated with `gfs` or `gdas`, we're OK.